pkg/kubectl/OWNERS

```diff
@@ -1,4 +1,4 @@
 approvers:
-- sig-cli-maintainers
```

Comment:
These seem like surprising aliases. I wouldn't expect these groups to correspond to approvers